cpp/unreal_plugins/SimulationController/Source/SimulationController/SimulationController.cpp

```diff
@@ -93,8 +88,14 @@ void SimulationController::postWorldInitializationEventHandler(UWorld* world, co
     if (!Config::s_initialized_) {
```

Comment:
Let's move `if (!Config::s_initialized_) return` to `StartupModule()` (immediately after asserting that the prerequisite modules are loaded), so a reader only needs to look at one function to understand how we turn off all the functionality in `SimulationController` if the `Config` system isn't loaded.